apps/web/src/services/iam/components/UserGroupManagementEditModal.vue

```diff
@@ -43,21 +56,32 @@ const handleCreate = () => {
 const handleUpdate = () => {
     console.log('TODO: Update API');
 };
+
+const fetchCreateUserGroup = async () => {
+    try {
+        await SpaceConnector.clientV2.identity.userGroup.create<UserGroupCreateParameters, UserGroupModel>({
+            name: state.groupName,
+            description: state?.description,
+        });
```

Comment:
👀: What happens after creating the data? Don't we need to reload the list?

Reply:
I've planned to add that logic after this PR.